### ISSUE TITLE
Missing or broken '@types/node' dependency

### ISSUE THREAD

**Issue by otri** [grpc/grpc-node#1538]
### Problem description
After adding the @google-cloud/pubsub module to my project, I got tsc compile errors:
```
node_modules/@google-cloud/pubsub/node_modules/@grpc/grpc-js/build/src/call.d.ts:68:5 - error TS2416: Property '_write' in type 'ClientWritableStreamImpl<RequestType>' is not assignable to the same property in base type 'Writable'.
  Type '(chunk: RequestType, encoding: string, cb: WriteCallback) => void' is not assignable to type '(chunk: any, encoding: string, callback: (err?: Error | undefined) => void) => void'.
    Types of parameters 'cb' and 'callback' are incompatible.
      Types of parameters 'err' and 'error' are incompatible.
        Type 'Error | null | undefined' is not assignable to type 'Error | undefined'.
          Type 'null' is not assignable to type 'Error | undefined'.

68     _write(chunk: RequestType, encoding: string, cb: WriteCallback): void;
       ~~~~~~

node_modules/@google-cloud/pubsub/node_modules/@grpc/grpc-js/build/src/call.d.ts:79:5 - error TS2416: Property '_write' in type 'ClientDuplexStreamImpl<RequestType, ResponseType>' is not assignable to the same property in base type 'Duplex'.
  Type '(chunk: RequestType, encoding: string, cb: WriteCallback) => void' is not assignable to type '(chunk: any, encoding: string, callback: (err?: Error | undefined) => void) => void'.
    Types of parameters 'cb' and 'callback' are incompatible.
      Types of parameters 'err' and 'error' are incompatible.
        Type 'Error | null | undefined' is not assignable to type 'Error | undefined'.
          Type 'null' is not assignable to type 'Error | undefined'.

79     _write(chunk: RequestType, encoding: string, cb: WriteCallback): void;
```

At this time, I did not have `@types/node` as a dependency.

Project built correctly after adding it, with:
```
npm install --save @types/node
```

### Environment
- MacOS 10.15.6 Catalina
- Node v10.18.1
- nvm use 10
- @google-cloud/pubsub 2.3.0

**Comment by murgatroid99** [comment]
I have published grpc-js 1.1.4 with a dependency on `@types/node`, which should fix this problem most of the time.

**Comment by otri** [comment]
Nice, thank you!
